projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -30,8 +31,8 @@
 class LandmarkFacialExpressionRecognitionNode:
 
     def __init__(self, input_image_topic="/usb_cam/image_raw",
```

Comment:
If this is indeed a topic for getting rgb images, the naming should change, as well as the argument's docstring name below.

```suggestion
    def __init__(self, input_rgb_image_topic="/usb_cam/image_raw",
```